src/pages/ApplicationsPage.tsx

```diff
@@ -1,21 +1,54 @@
 import React from 'react';
 import { Helmet } from 'react-helmet';
-import { PageSection, PageSectionVariants } from '@patternfly/react-core';
+import { useK8sWatchResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { Bullseye, PageSection, PageSectionVariants, Spinner } from '@patternfly/react-core';
 import ApplicationDetailsView from '../components/ApplicationDetailsView/ApplicationDetailsView';
 import ApplicationListView from '../components/ApplicationListView/ApplicationListView';
-import NamespacedPage from '../components/NamespacedPage/NamespacedPage';
+import { GettingStartedCard } from '../components/GettingStartedCard/GettingStartedCard';
+import { GettingStartedModal } from '../components/modal/GettingStartedModal';
+import NamespacedPage, { useNamespace } from '../components/NamespacedPage/NamespacedPage';
 import PageLayout from '../components/PageLayout/PageLayout';
 import HacbsApplicationDetails from '../hacbs/components/ApplicationDetails/HacbsApplicationDetails';
 import { useQuickstartCloseOnUnmount } from '../hooks/useQuickstartCloseOnUnmount';
+import imageUrl from '../imgs/getting-started-illustration.svg';
+import { ApplicationGroupVersionKind } from '../models';
 import { getQueryArgument } from '../shared/utils';
+import { ApplicationKind } from '../types';
+
+const GETTING_STARTED_MODAL_KEY = 'application-list-getting-started-modal';
+const GETTING_STARTED_CARD_KEY = 'application-list-getting-started-card';
 
 const ApplicationsPage = () => {
   useQuickstartCloseOnUnmount();
   const applicationName = getQueryArgument('name');
   const hacbs = JSON.parse(getQueryArgument('hacbs'));
 
+  const { namespace } = useNamespace();
+
+  const [applications] = useK8sWatchResource<ApplicationKind[]>({
```

Comment:
If you're fetching all the applications here then maybe send it to the `ApplicationListView` as well.